docs/shadow_config_spec.md

```diff
@@ -661,6 +662,33 @@ environment:
 environment: { ENV_A: "1", ENV_B: foo }
 ```
 
+#### `hosts.<hostname>.processes[*].expected_final_state`
```

Comment:
Maybe also double-check other documentation examples (for example "docs/getting_started_basic.md") and make sure those examples are updated to use this as well.